extensions/components/com_redcore/site/models/translation.php

```diff
@@ -0,0 +1,315 @@
+<?php
```

Comment:
I assume that in order to use the admin model's save method, I'd then have to create a save method for the controller, that specifically uses the admin model instead?